frontend/src/queries/query.ts

```diff
@@ -15,7 +13,9 @@ import {
     PersonsNode,
     QueryStatus,
     RefreshType,
-} from './schema'
+} from '~/queries/schema/schema-general'
```

Comment:
This is unrelated to the actual changes here and just gets rid of an error message related to the import.